services/src/Microsoft.Azure.IIoT.Services.Processor.Telemetry.Ux/src/Runtime/Config.cs

```diff
@@ -0,0 +1,58 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.Azure.IIoT.Services.Processor.Telemetry.Ux.Runtime {
+    using Microsoft.Azure.IIoT.Diagnostics;
+    using Microsoft.Azure.IIoT.Hub.Processor;
+    using Microsoft.Azure.IIoT.Hub.Processor.Runtime;
+    using Microsoft.Azure.IIoT.Messaging.EventHub;
+    using Microsoft.Azure.IIoT.Messaging.EventHub.Runtime;
+    using Microsoft.Azure.IIoT.Messaging.SignalR;
+    using Microsoft.Azure.IIoT.Messaging.SignalR.Runtime;
+    using Microsoft.Extensions.Configuration;
+    using System;
+
+    /// <summary>
+    /// Telemetry processor service configuration
+    /// </summary>
+    public class Config : DiagnosticsConfig, IEventProcessorConfig, 
+        IEventHubConsumerConfig, ISignalRServiceConfig {
+
+        /// <inheritdoc/>
+        public string EventHubConnString => _eh.EventHubConnString;
+        /// <inheritdoc/>
+        public string EventHubPath => _eh.EventHubPath;
+        /// <inheritdoc/>
+        public string ConsumerGroup => "telemetryux";
```

Comment:
same as above? should probably be a parm in config as its defined in the arm template?

Reply:
In this case they are hardcoded, they always connect to the same consumer group.   You just have to make sure such group is defined.